jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseProjectServiceImpl.java

```diff
@@ -83,12 +83,16 @@ public void configureNewCaseProject(WorkspaceProject project) {
         final String separator = Paths.convert(rootPath1).getFileSystem().getSeparator();
 
         // add empty .caseproject file as marker
-        String rootPathString = project.getRootPath().toURI().toString() + separator + CASE_PROJECT_DOT_FILE;
-        Path rootPath = ioService.get(URI.create(rootPathString));
-        ioService.write(rootPath,
-                        "");
-        logger.debug("Added caseproject marker (dot) file at {}",
-                     rootPath);
+        try {
+            String caseProjectDotFilePath = project.getMainModule().getRootPath().toURI().toString() + CASE_PROJECT_DOT_FILE;
+            ioService.write(ioService.get(new URI(caseProjectDotFilePath)),
```

Comment:
@tsurdilo perhaps test if marker file already exists, similar to what is done for the deployment descriptor file?